cluster-autoscaler/cloudprovider/hetzner/hetzner_cloud_provider.go

```diff
@@ -179,6 +181,29 @@ func (d *HetznerCloudProvider) Refresh() error {
 	return nil
 }
 
+// Check if any defined placement groups could potentially have more than the maximum allowed number of nodes
+func getLargePlacementGroups(nodeGroups map[string]*hetznerNodeGroup, threshold int) []hcloud.PlacementGroup {
+	placementGroupTotals := make(map[hcloud.PlacementGroup]int)
```

Comment:
Pointer needed for hcloud.PlacementGroup.
```suggestion
func getLargePlacementGroups(nodeGroups map[string]*hetznerNodeGroup, threshold int) []*hcloud.PlacementGroup {
	placementGroupTotals := make(map[*hcloud.PlacementGroup]int)
```